README.md

```diff
@@ -30,3 +45,46 @@ within the application.**
 
 Currently, the library is developed with Postgres as the target DB. But the approach is applicable to any DB supporting stored procedure/functions – Oracle, MS-SQL, ...
 
+
+### Usage
+
+#### Sbt
+
+```scala
+libraryDependencies ++= Seq(
+  "za.co.absa.fa-db" %% "core"  % "X.Y.Z",
+  "za.co.absa.fa-db" %% "slick" % "X.Y.Z"
+)
+```
+
+#### Maven
+
+##### Scala 2.11 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa/fa-db_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/za.co.absa/fa-db_2.11)
+
```

Comment:
```suggestion
### Scala 2.11 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa.fa-db/core_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/za.co.absa.fa-db/core_2.11)

```

If we want to be technically correct, then you should probably include badges/links both to core & slick artifacts. But for the general sense, this is probably fine (expected that both artifacts would be available always together).

I just fixed the links.